sdk/servicebus/service-bus/test/public/sendAndSchedule.spec.ts

```diff
@@ -61,11 +61,19 @@ describe("Sender Tests", () => {
       : TestMessage.getSample();
     await sender.sendMessages(testMessage);
     const msgs = await receiver.receiveMessages(1);
+    // remove message first in case any assertion fails to ensure we don't have lingering message
+    await receiver.completeMessage(msgs[0]);
 
     should.equal(Array.isArray(msgs), true, "`ReceivedMessages` is not an array");
     should.equal(msgs.length, 1, "Unexpected number of messages");
     should.equal(msgs[0].deliveryCount, 0, "DeliveryCount is different than expected");
-    should.equal(msgs[0].state, "active");
+    should.equal(msgs[0].messageId, testMessage.messageId);
+    // the message could be in "scheduled" state because we set `scheduledEnqueueTimeUtc` property
+    should.equal(
+      msgs[0].state === "active" || msgs[0].state === "scheduled",
+      true,
+      `actual message state: ${msgs[0].state}`
+    );
```

Comment:
NIT: perhaps use `isTrue` instead of `equal`? also `msg[0]` could be pulled into a variable since it is referenced multiple times.

Reply:
The BDD style is used here. But yeah I can come up with a better assertion.

Reply:
changed to `(...).should.be.true;` I lost the ability to specify a custom message but it's fine.

Reply:
had to disable linting for unused expression :(

Reply:
my suggestion on using a truethness check seems to complicate things so it is fine to revert it.